src/shared/components/common/loading-skeleton.tsx

```diff
@@ -1,43 +1,46 @@
-import { setIsoData } from "@utils/app";
-import dataBsTheme from "@utils/browser/data-bs-theme";
 import { Component } from "inferno";
 
 interface LoadingSkeletonProps {
   itemCount?: number;
 }
 
-interface LoadingSkeletonItemProps {
-  theme: string;
+interface LoadingSkeletonLineProps {
+  size: 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
```

Comment:
Seems like just `number` would be better.

Reply:
it's because I'm trying to cover the rage of the bootstrap col class, it goes from 1 to 12. If I put number, I'll allow someone to put something bigger than 12 or smaller than 1 in the future.

Reply:
Ah, its very minor and ultimately up to you, just seemed a little overkill. You can leave as is if you like.